app/templates/admin/permissions/system-roles.hbs

```diff
@@ -46,10 +46,10 @@
       </td>
       <td>
         <div class="ui horizontal compact basic buttons">
-          {{#ui-popup content=(t 'Edit') class='ui icon button' position='left center'}}
+          {{#ui-popup content=(t 'Edit') class='ui icon button'}}
             <i class="edit icon"></i>
           {{/ui-popup}}
-          {{#ui-popup content=(t 'Delete') class='ui icon button' position='left center'}}
```

Comment:
@utkarshnath That makes sense, should have had been noticed in PR #240 itself, approving these then.